docs/samples/iot_hub_state_export.json

```diff
@@ -0,0 +1,554 @@
+                                    "endpoint1"
+                                ],
+                                "isEnabled": false,
+                                "name": "route2",
+                                "source": "DeviceMessages"
+                            }
+                        ]
+                    },
+                    "storageEndpoints": {
+                        "$default": {
+                            "authenticationType": "keyBased",
+                            "connectionString": "DefaultEndpointsProtocol=https;EndpointSuffix=core.windows.net;AccountName=teststore;AccountKey=FAKEKEY;BlobEndpoint=https://teststore.blob.core.windows.net/;FileEndpoint=https://teststore.file.core.windows.net/;QueueEndpoint=https://teststore.queue.core.windows.net/;TableEndpoint=https://teststore.table.core.windows.net/",
+                            "containerName": "test-hub-container",
+                            "sasTtlAsIso8601": "PT1H"
+                        }
+                    }
+                },
+                "sku": {
+                    "capacity": 1,
+                    "name": "S1",
+                    "tier": "Standard"
+                },
+                "tags": {
+                    "environment": "test"
+                },
+                "type": "Microsoft.Devices/IotHubs"
+            },
+            {
+                "apiVersion": "2022-04-30-preview",
+                "dependsOn": [
+                    "[resourceId('Microsoft.Devices/IotHubs', 'test-hub')]"
+                ],
+                "name": "test-hub/certificate1",
+                "properties": {
+                    "certificate": "CERTIFICATEDATA",
+                    "isVerified": false
+                },
+                "type": "Microsoft.Devices/IotHubs/certificates"
+            }
+        ],
+        "variables": {}
+    },
+    "configurations": {
+        "admConfigurations": {
+            "my_config": {
+                "content": {
+                    "deviceContent": {
+                        "properties.desired.awesome": 3
+                    }
+                },
+                "id": "my_config",
+                "labels": null,
+                "metrics": {
+                    "queries": {},
+                    "results": {}
+                },
+                "priority": 10,
+                "systemMetrics": {
+                    "queries": {
+                        "appliedCount": "select deviceId from devices where configurations.[[my_config]].status = 'Applied'",
+                        "targetedCount": "select deviceId from devices where tags.environment='test'"
+                    },
+                    "results": {
+                        "appliedCount": 0,
+                        "targetedCount": 0
+                    }
+                },
+                "targetCondition": "tags.environment='test'"
+            }
+        },
+        "edgeDeployments": {
+            "deployment1": {
```

Comment:
Are layered edge deployments supported?

Reply:
they should be - I will need to double check

Reply:
the layered deployments are used